charters/DocumentationCharter.md

```diff
@@ -0,0 +1,49 @@
+# Documentation Working Group Charter
+
+## Mission and Goals
+
+The Documentation Working Group ("Docs Working Group") serves as a support, helper, and advisory body to Jupyter Subprojects on all aspects of documentation. The core pillars of our mission:
+
+- Improve all aspects of documentation across the Jupyter ecosystem.
+- Make high quality documentation that is clear, comprehensive, inclusive, and serves the varying needs of Jupyter's diverse community.
+- Engage with the community to help users with Jupyter products, discover their needs, and connect them with relevant information, expertise and resources.
+
+The Docs Working Group will provide a place for consistent, focused, holistic efforts to be spent on documentation across the whole ecosystem. This group exists in part to provide capacity and resources to the subprojects (some of which are already suffering from a lack of resources/capacity, and more specifically to work on documentation in particular).
+
+We want all users to have positive experiences inside the Jupyter ecosystem, especially users who are learning and coming in for the first time, and users with disabilities.
+
+## Areas of Responsibility
+
+The Docs Working Group is responsible for the following:
+
+- Setting standards for documentation where needed.
```

Comment:
Are standards "binding" for sub-projects or are they more like shared resources that sub-projects can use if they wish?

An example - in the JupyterHub project we have a shared modification of the PyData Sphinx Theme defined here:

https://github.com/jupyterhub/jupyterhub-sphinx-theme

This allows JupyterHub projects to use the theme in order to standardize the look and feel across sub-projects. But it's not a "requirement" - it's just a shared resource to be used as is useful.

Reply:
The vision I have for this: The PyData Sphinx theme could be further developed by the Docs Working Group, in consultation/collaboration with the JupyterHub team and existing devs, then we could take point on advocating for it, both as a resource to teams that need it, and as a standard to make things consistent across the ecosystem.

This collaborative pattern is at the heart of how the Docs Working Group will operate (we help, we do work, we collaborate to understand needs and get feedback, we build resources that other teams can use, and we advocate for their usage to subprojects in accordance with our mission to make the Jupyter ecosystem better and more consistent).

If we do our work right, we will have gathered needs from the subprojects that inform what the theme is capable of, so that they have what they need for that resource to make sense for their project.

Reply:
I'd love to see this working group be a source of more focused development (or even just specific requests) in the PyData theme (or the [sphinx book theme](https://github.com/executablebooks/sphinx-book-theme)). Those are both chronically under-supported and maintainer-led projects!